### PR TITLE
Don't mangle i18n

### PR DESCRIPTION
**Pull request by kevinbarabash**
## Summary:
In order for build_mobile_perseus.py to extract strings all of the translation functions need to start with 'i18n.'.  If we rename this boject it won't be able to extract those strings.  This PR fixes this by excluding 'i18n' from mangle.  The terser config is almost identical to the one in webapp.

I've also replace the circle-ci build to run `yarn test` with a GitHub Actions check that does the same.

Issue: FEI-3478

## Test plan:
- yarn build
- inspect build/math-input.js, see that 'i18n._()' calls abound
- yarn link
- cd ../webapp/services/static
- yarn link "@khanacademy/math-input"
- ./content_render/tools/build_mobile_perseus.py 
- see that it completes and that khan/mobile had an updated perseus bundle and template files
- rebuild and run the iOS app in Xcode, see that exercises load and work correctly

### CODE REVIEW

**Comment by kevinbarabash** [comment]
Please ignore the failing `CircleCI` job.

**Comment by kevinbarabash** [comment]
I somehow broke the storybook build that gets deployed to netlify so I'll have to fix that before this can proceed.

**Comment by kevinbarabash** [comment]
I see, it's only some of the stories that are broken in storybook.

**Comment by kevinbarabash** [comment]
The storybook is an existing issue on http://khan-math-input.netlify.app so I'm going to punt on it for now.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -33,3 +33,33 @@ jobs:
...
+          node-version: 12.x
+      - name: Get yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
```
Comment (jeresig) [inline .github/workflows/nodejs.yml:54]: I don't think you need this - caching node modules (below) is sufficient.

**Review by jeresig — APPROVED** [review]
Great, thank you for fixing this!

**Pull request merged**